csrc/torch_bindings.cpp

```diff
@@ -133,6 +133,15 @@ TORCH_LIBRARY_EXPAND(TORCH_EXTENSION_NAME, ops) {
   ops.def("gptq_marlin_24_gemm", &gptq_marlin_24_gemm);
   ops.impl("gptq_marlin_24_gemm", torch::kCUDA, &gptq_marlin_24_gemm);
 
+  // Machete (Dense) Optimized Mixed Precision GEMM for Hopper.
+  ops.def("machete_supported_schedules", &machete::supported_schedules);
+  ops.impl("machete_supported_schedules", torch::kCPU,
+           &machete::supported_schedules);
```

Comment:
this `impl` should be redundant when using `def` w/a function pointer.

Reply:
cool, removed